README.md

```diff
@@ -176,22 +176,25 @@ object Nel {
 ### Generic Instances
 
 In some cases, you want to generically provide instances for a certain type class for
-all new types based of a certain type. You can take advantage of the `Coercible`
-type class to achieve that. Here is an example of providing `Numeric` instances
-to all new types of `Int`
+all new types based. You can take advantage of the `Coercible`
+type class to achieve that.
 
-```scala
-scala> implicit def coercibleNumericInt[A](implicit ev: Coercible[Int, A]): Numeric[A] =
-         implicitly[Numeric[Int]].asInstanceOf[Numeric[A]]
+Here is an example to provide `Eq` instances for all new type using `Eq` instances of base type. 
 
+```scala
+scala> implicit def coercibleEq[A, B](implicit ev: Coercible[Eq[A], Eq[B]], A: Eq[A]): Eq[B] =
+    A.asInstanceOf[Eq[B]]
```

Comment:
Why the cast? I think you can do `ev(A)`.

Reply:
I think the JIT can inline the `ev.apply(A)`, and it's arguably better to prefer the `ev.apply` method as it's type safe, whereas `asInstanceOf` is not.